### ISSUE TITLE
Multiple Path-Matches

### ISSUE THREAD

**Issue by TheBlueMatt** [nttgin/BGPalerter#317]
I use an old hacked-up version of the python BGPAlerter to monitor for TE correctness as well as hijacks. This needs to capture both, for example,
 * AS1 must only be visible via path ".*(2|3|4)1$", and,
 * when a peer sees it transiting 2914, must transit AS3 (ie match "2914" nomatch ".*2914,3,1$").

I admit its entirely possible its possible to capture this with the existing regex logic, but its certainly not trivial or clear to me how to do that, and its rather trivial to do via two patch match attributes.

**Comment by muffizone** [comment]
I'll admit I have been trying to hack my way around regexes for the first option but without any success.

So bumping this request for visibility.

**Comment by massimocandela** [comment]
I added a feature which allows to specify multiple matching rules.
This makes it easier and allows for different `matchDescription` (reported in the alert).

I also created a dedicated page with some examples [here](https://github.com/nttgin/BGPalerter/blob/dev/docs/path-matching.md) (it is a draft, errors may be present).

It will go out with the next release, for now it's in dev. I tested it, but another pair of eyes is always better.
